Neos.Neos/Resources/Private/Fusion/Prototypes/MenuDataItem.fusion

```diff
@@ -0,0 +1,44 @@
+prototype(Neos.Neos:MenuDataItem) < prototype(Neos.Fusion:Component) {
+  currentDocumentNode = ${documentNode}
+  menuItemNode = ${this.menuItemNode}
```

Comment:
I suggest `menuItemNode = null`, because like that you reference the property to itself